README.md

```diff
@@ -1 +1,10 @@
-# bus-mall
\ No newline at end of file
+# bus-mall
+
+project description: 
+make a thing that'll cycle through 3 images at a time. when you click on one it'll increase vote count and produce 3 images, none of which are supposed to have been in the group you were just previously viewing. 
+
+
+3/19 Updates:
+Got scaffolding set up, images cycling with no repeats then displaying results afterwards. got some help with Justin & Jason, and lots of advice from TAs. Worked with Jason & Stephan on for the wireframe.html which I only just afterwards realized should just be implemented into the index.html. Got to wireframe portion at the of the lab, having trouble merging it into index. Most of it should be carried over, but not all is displaying as I'd like. 
+
+Have Temp coloring in css to help align sections. will be commented out for submission
```

Comment:
Lets start making the README less of a work/todo list and more of the documentation for how the end user should expect to use the app.